src/ui/dialogs/avatarupload.ts

```diff
@@ -0,0 +1,177 @@
+import Dialog from '../Dialog'
+import Client from '../../Client'
+
+let avatarUploadTemplate = require('../../../template/avatarUploadTemplate.hbs');
+
+let dialog: Dialog;
+let base64data : string;
+let mimetype : string;
+
+let height: string;
+let width: string;
+
+let account;
+
+export default function() {
+
+   let content = avatarUploadTemplate();
+
+   dialog = new Dialog(content);
+   let dom = dialog.open();
+
+   let accounts = Client.getAccountManager().getAccounts();
+   for (let i=0;i<accounts.length;i++){
+       let accountoption = $('<option value="'+i+'">'+accounts[i].getUid()+'</option>');
+       dialog.getDom().find('select[name="account"]').append(accountoption);
+   }
```

Comment:
I see the use cases that you like to have different avatars for your different accounts, but how can we visualize that in the roster?

Reply:
you mean the presence status bar on the bottom? good question... 
1. we could only show the first's account avatar there and serve the other avatars only for the contacts...
2. show a litte animation and switch between all avatars
3. show the first avatar and while hovering switch between others or show a little box with all others...
other ideas?